pkg/backend/openshiftcluster.go

```diff
@@ -102,7 +101,7 @@ func (ocb *openShiftClusterBackend) handle(ctx context.Context, log *logrus.Entr
 		return err
 	}
 
-	hiveRestConfig, err := hive.HiveRestConfig()
+	hiveRestConfig, err := ocb.env.LiveConfig().HiveRestConfig(ctx, 1)
 	if err != nil {
 		log.Info(err) // TODO(hive): Update to fail once we have Hive everywhere in prod and dev
```

Comment:
Its likely we could get AKS everywhere before this is merged. Either way can we not log an error here now? 

Also if there is an error getting MSI credentials inside `HiveRestConfig` we will we retry this?

Reply:
I think this is out of scope for this PR. 

As far as retrying the MSI, there's a lot of other places it will crash first.